locale/en.json

```diff
@@ -583,6 +587,7 @@
   "TERMS_CHECKBOX_2": "terms and conditions",
   "TERMS_AND_CONDITIONS": "Terms and Conditions",
   "TERMS_ERROR": "You must accept the terms and conditions.",
+  "BACK_TO_SIGHTING": "Back to sighting",
```

Comment:
Just a general note that I try to add new entries to the bottom of this list just in case there's a merge conflict with someone else... I think it'll handle conflicting code better if it's all in the same narrower area.